site/en/guide/estimator.ipynb

```diff
@@ -831,20 +819,22 @@
       },
       "source": [
         "<a name=\"estimator_support\"></a>\n",
+        "\n",
         "### What's supported now?\n",
         "\n",
         "There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There may also be a number of bugs in this integration. At this time, we do not plan to actively improve this support, and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to use `tf.distribute` with those APIs instead.\n",
```

Comment:
```suggestion
        "There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There may also be a number of bugs in this integration and there are no plans to improve this support, and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to use `tf.distribute` with those APIs instead.\n",
```

remove the last "we" :)

Reply:
Good catch, thanks @lamberta. One more thing:

```diff
There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training
and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There
- may also be a number of bugs in this integration and there are no plans to actively improve this support, 
+ may also be a number of bugs in this integration and there are no plans to actively improve this support 
- and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to
+ (the focus is on Keras and custom training loop support). If at all possible, you should prefer to
use `tf.distribute` with those APIs instead.\n",
```

WDYT?

Reply:
That works. Thanks!